### ISSUE TITLE
Feature Request: Multiple bib files in one bibliography

### ISSUE THREAD

**Issue by wrenger** [typst/typst#293]
First of all, I really like typst and have already played a bit with styling and scripting (creating themes and rebuilding a latex acronym package).

The feature I'm missing most for writing papers is the ability to add multiple bib files. For example, we have separate bib files for our own and external references that are composed into a single bibliography.

Something like the following would be great:

```typ
#bibliography(("bib/own.bib", "bib/external.bib"), title: "References", style: "ieee")
```

This would just combine the two bib files, returning errors on duplicate keys as usual.

**Comment by wrenger** [comment]
I made a PR for this: https://github.com/typst/typst/pull/296

**Comment by laurmaedje** [comment]
Fixed in #296.
